demos/PersonalizerTravelAgencyDemo/PersonalizerTravelAgencyDemo/wwwroot/js/demo.js

```diff
@@ -147,9 +147,19 @@ document.addEventListener("DOMContentLoaded", function () {
     articleViewer.addEventListener("load", function () {
         const articleDoc = articleViewer.contentDocument;
         const mainContainer = articleViewer.contentWindow.document.getElementById("main-container");
-        const articleFooter = articleViewer.contentWindow.document.getElementById("article-footer");
         const gauge = articleViewer.contentWindow.document.getElementById("gauge");
         const boundSetIframeContentSize = setIframeContentSize.bind(null, mainContainer);
+        const modalButton = articleViewer.contentWindow.document.getElementById('endModal-close-button');
+        const modalIcon = articleViewer.contentWindow.document.getElementById('endModal-close-icon');
+        const checkoutModal = articleViewer.contentWindow.document.getElementById('modal-checkout');
+
+        modalButton.addEventListener('click', function () {

```

Comment:
Use the function reference instead of creating a new function.
```suggestion
        modalButton.addEventListener('click', goToHomeSite);
```